### DISCUSSION TITLE
Your account has been flagged...(((

### DISCUSSION THREAD

**Discussion by ViktorSvertoka** [community/community#52893] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
tell me what to do next, I wrote to the support service but for a week there is no answer ...

And I need an account for studying, I hand over my homework at the academy through it.

It was blocked 7 days ago due to the fact that I tried to log in to www.netlify.com more than 5 times and after an unsuccessful authorization attempt, I was blocked by github😭😭😭

**Comment by HardlinePizza21** [comment]
How do you solve this problem.
